.gitignore

```diff
@@ -9,6 +9,7 @@ npm-debug.*
 *.orig.*
 web-build/
 web-report/
+package-lock.json
```

Comment:
Good idea. Can you move this to a separate section and add a one-line comment along the lines of:
`Avoid accidentally commiting npm package-lock file - we are using yarn`